### DISCUSSION TITLE
underline a text in richeditor

### DISCUSSION THREAD

**Discussion by aliloubm** [filamentphp/filament#921] [Help] [answered]
is it possible to add an option to add an "underline" button in RichEditor ?

**Comment by ryangjchandler** [comment]
@aliloubm The `RichEditor` field uses Trix so we're quite limited in what we can do ourselves.

**Reply by MACscr** [reply]
Here is a link on how to easily add the underline button to Trix https://codepen.io/javan/pen/EPqzZo

**Reply by ryangjchandler** [reply]
Thanks @MACscr.

**Comment by ryangjchandler** [comment, marked as answer]
@aliloubm Please see the response from @MACscr.

> Here is a link on how to easily add the underline button to Trix [codepen.io/javan/pen/EPqzZo](https://codepen.io/javan/pen/EPqzZo).
